compute_endpoint/packaging/debian/rules

```diff
@@ -15,25 +15,20 @@ export DEB_BUILD_OPTIONS
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 name := globus-compute-endpoint
-versionedname := $(name)
-VIRTUAL_ENV := /opt/globus
+VIRTUAL_ENV := /opt/$(name)/venv-py39
 TMP_BUILD_DIR := debian/$(name)-tmp
 TMP_VIRTUAL_ENV := $(TMP_BUILD_DIR)$(VIRTUAL_ENV)
-DEST_ROOT := debian/$(versionedname)
+DEST_ROOT := debian/$(name)
 DEST_VIRTUAL_ENV := $(DEST_ROOT)$(VIRTUAL_ENV)
+PYTHON3 := /opt/globus-python/bin/python3
 
 _sysconfdir=/etc
 _sbindir=/usr/sbin
-_mandir=/usr/share/man
-_var=/var
 _unitdir=/lib/systemd/system
-_docdir=/usr/share/doc
-_httpd_confdir=$(_sysconfdir)/apache2/conf-available
-_completion_dir="${_sysconfdir}/bash_completion.d"
 
 .PHONY: override_dh_auto_configure
 override_dh_auto_configure:
-	/opt/globus-python/bin/python3 -mvenv $${PWD}/$(TMP_VIRTUAL_ENV)
+	$(PYTHON3) -mvenv $${PWD}/$(TMP_VIRTUAL_ENV)
```

Comment:
Any particular reason to use `-mvenv` syntax instead of the more common `-m venv`?